chat-plugins/info.js

```diff
@@ -202,7 +203,10 @@ exports.commands = {
 				}
 			});
 		}
-		if (!results.length) return this.errorReply("No results found.");
+		if (!results.length) {
+			let invalidTarget = net.isIP(target.trim()) === 0 && !target.includes('.');
+			return this.errorReply("No results found" + (invalidTarget ? " because this is not a valid IP or host search quarry." : "."));
```

Comment:
I've personally never heard "quarry" being used to mean "object of search" before, but it's in [dictionaries](https://en.wiktionary.org/wiki/quarry#Etymology_2). Most likely, its use is restricted to certain regions, and/or is rare (especially outside of written documents)

Definitely prefer to use a less obscure word here, such as "target" or "query"